Engine/Fuego/Windows/WindowWin.cpp

```diff
@@ -5,6 +5,66 @@
 
 namespace Fuego
 {
+
+DWORD __stdcall WindowWin::WinThreadMain(LPVOID lpParameter)
```

Comment:
Shouldn't it be some Win32 macro instead of __stdcall?

Reply:
No point to be in the header